src/test/resources/fixtures/community-api/assessments.json

```diff
@@ -1,4 +1,4 @@
 {
   "rsrScore": 23.0,
-  "ogrsScore": 16
+  "ogrsScore": 21
```

Comment:
I just changed this so it made the needs work testable via the entry point. If this is not a valid OGRS score I will have to find another way to score 2 points

Reply:
Should be fine.